components/formats-gpl/src/loci/formats/in/MetamorphTiffReader.java

```diff
@@ -73,9 +73,9 @@
   // -- Fields --
 
   private String[] files;
-  private int wellCount = 0;
-  private int fieldRowCount = 0;
-  private int fieldColumnCount = 0;
+  private int wellCount = 1;
```

Comment:
I assume there is no downside to changing the default values here since these are private and not introspectable. The only thing I could think of would be some logic in the reader that would rely on the initialization state of these variables i.e.`(if wellCount == 0)`.

Reply:
There are no checks currently to ensure the values are initialised. If they haven't been set correctly then you actually end up with other issues such as 0 image count and divide by zero issues. I can see how it might be beneficial to log a warning in the case when we haven't initialised them and are falling back to the default though.